data/io.elementary.appcenter.gschema.xml

```diff
@@ -41,5 +41,10 @@
       <summary>Unix UTC time of last cache refresh</summary>
       <description>Used to determine when AppCenter last refreshed its caches and checked for package updates</description>
     </key>
+    <key type="b" name="automatically-install-flatpak-updates">
+      <default>false</default>
+      <summary>Automatically install Flatpak updates</summary>
+      <description>Whether to automatically install updates to curated Flatpaks</description>
+    </key>
```

Comment:
Series of nits. I know your name is more precise, but it's pretty wordy, when a more concise key would do—especially as I don't see us offering multiple different "automatic updates" mechanisms inside of AppCenter in the future. This would also more closely match the copy used in-app, which is convenient.

```suggestion
    <key type="b" name="automatic-updates">
      <default>false</default>
      <summary>Automatically install updates</summary>
      <description>Whether to automatically install updates to curated Flatpak apps</description>
    </key>
```